src/Admin/PageAdmin.php

```diff
@@ -255,16 +261,16 @@ protected function configureFormFields(FormMapper $form): void
             $form
                 ->with('main')
                     ->add('parent', PageSelectorType::class, [
-                        'page' => $page ?: null,
-                        'site' => $page ? $page->getSite() : null,
+                        'page' => $page,
+                        'site' => null !== $page ? $page->getSite() : null,
```

Comment:
```suggestion
                        'site' => $page?->getSite(),
```

Reply:
nullable operations are php 8 Afaik

Reply:
Oh ! My bad, I thought we dropped 7.4 on this branch. Sorry.